docs/development/pigs/pig-001.rst

```diff
@@ -0,0 +1,120 @@
+.. include:: ../../references.txt
+
+.. _pig-001:
+
+**********************************
+PIG 1 - PIG purpose and guidelines
+**********************************
+
+* Author: Christoph Deil
+* Created: December 20, 2017
+* Status: draft
```

Comment:
Maybe add a reference to the initial pull request for the PIG, where most of the discussion took place?

Reply:
ok, will do.

Reply:
done